code/datums/antagonists/abductor.dm

```diff
@@ -72,11 +79,49 @@
 	var/datum/species/abductor/A = H.dna.species
 	A.scientist = TRUE
 
+/datum/antagonist/abductor/admin_add(datum/mind/new_owner,mob/admin)
+	var/list/current_teams = list()
+	for(var/datum/team/abductor_team/T in get_all_teams(/datum/team/abductor_team))
+		current_teams[T.name] = T
+	var/choice = input(admin,"Add to which team ?") as null|anything in current_teams + "new team"
+	if(choice in current_teams)
+		team = current_teams[choice]
+	else if (choice == "new team")
+		team = new
+	else
+		return
+	new_owner.add_antag_datum(src)
+	log_admin("[key_name(usr)] made [key_name(new_owner.current)] [name] on [choice]!")
+	message_admins("[key_name_admin(usr)] made [key_name_admin(new_owner.current)] [name] on [choice] !")
+
+/datum/antagonist/abductor/get_admin_commands()
+	. = ..()
+	.["Equip"] = CALLBACK(src,.proc/admin_equip)
+
+/datum/antagonist/abductor/proc/admin_equip(mob/admin)
+	if(!ishuman(owner.current))
+		to_chat(admin, "<span class='warning'>This only works on humans!</span>")
+		return
+	var/mob/living/carbon/human/H = owner.current
+	var/gear = alert(admin,"Agent or Scientist Gear","Gear","Agent","Scientist")
+	if(gear)
+		if(gear=="Agent")
+			H.equipOutfit(/datum/outfit/abductor/agent)
+		else
+			H.equipOutfit(/datum/outfit/abductor/scientist)
 
 /datum/team/abductor_team
 	member_name = "abductor"
 	var/team_number
 	var/list/datum/mind/abductees = list()
+	var/static/team_count = 1
+	var/const/max_teams = 4
```

Comment:
I'd prefer defs to const because we have proven that this does create another real var sadly